pages_bots/ncc_page.py

```
@@ -0,0 +1,114 @@
+        print(s)
+
+
+catdepth_new.SITECODE = "www"
+catdepth_new.FAMILY = "nccommons"
+
+
+dir2 = os.getenv("HOME")
+printt(f"HOME:{dir2}")
+# ---
+if not dir2:
+    Dir = str(Path(__file__).parents[0])
```

Pylint (3.3.7)

[convention] Constant name "Dir" doesn't conform to UPPER_CASE naming style

(C0103)